tests/test_openvpn.py

```diff
@@ -67,6 +67,12 @@ def test_json_dict_tap(self):
     def test_empty_string(self):
         OpenvpnParser(data='{}')
 
+    def test_random_string(self):
+        OpenvpnParser(data='{string123}')
+
+    def test_bogus_data(self):
+        OpenvpnParser(data='{%$^*([[zsde4323@#}')
```

Comment:
we expect an exception to be raise here, this test cannot pass like that, you have to do something like here:
https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/tests/test_config.py#L22-L28

**Note** I already explained you how to do this via chat. I hope that with this more explicit comment you now understand.